src/main/java/com/aws/greengrass/deployment/bootstrap/BootstrapManager.java

```diff
@@ -429,6 +467,18 @@ public void persistBootstrapTaskList(Path persistedTaskFilePath) throws IOExcept
         logger.atInfo().kv("filePath", persistedTaskFilePath).log("Bootstrap task list is saved to file");
     }
 
+    /**
+     * Delete the bootstrap task list file, if it exists.
+     *
+     * @param persistedTaskFilePath Path to the persisted file of bootstrap tasks
+     * @throws IOException on I/O error
+     */
+    public void deleteBootstrapTaskList(Path persistedTaskFilePath) throws IOException {
+        Objects.requireNonNull(persistedTaskFilePath);
```

Comment:
Require throws an exception, is that desired? Should this just log and then return perhaps?

Reply:
NullPointerException is not desirable, I will change this